src/ensembl/src/shared/components/expandable-section/ExpandableSection.tsx

```diff
@@ -70,7 +66,11 @@ const ExpandableSection = (props: ExpandableSectionProps) => {
     <div className={wrapperClassNames}>
       {props.expandedContent && (
         <div className={styles.toggle} onClick={toggleExpanded}>
-          <Chevron className={chevronClasses} />
+          <Chevron
+            direction={isExpanded ? 'up' : 'down'}
+            animateDirectionChange={true}
```

Comment:
as said earlier `animate={true}` sounds better. The only animation we do here is a direction change.

Reply:
✅